README.md

```diff
@@ -18,6 +18,7 @@ A "toolkit" is a set of tools in any form that allows you to create applications
 * [Create React App](https://github.com/facebookincubator/create-react-app) - Create React apps with no build configuration.
 * [Create React App Typescript](https://github.com/wmonk/create-react-app-typescript) - Create React apps using typescript with no build configuration.
 * [vue-cli](https://github.com/vuejs/vue-cli) - A simple CLI for scaffolding Vue.js projects.
+* [preact-cli](https://github.com/developit/preact) - High-performance Preact PWA's in seconds.
```

Comment:
I think the link should be https://github.com/developit/preact-cli, shouldn't it?

Reply:
Yup, my bad!